blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,79 @@ class EmberElectronBlueprint extends Blueprint {
   afterInstall(/* options */) {
     let logger = new Logger(this);
 
-    return this._installDependencies(logger)
-      .then(() => this._createResourcesDirectories())
-      .then(() => this._ensurePackageJsonConfiguration())
-      .then(() => this._updateGitignore())
```

Comment:
Will be removing this again, as outDir is supported upstream + we have bo other ignores.